### PR TITLE
Move all logging to use static Logging class.

### PR DESCRIPTION
**Pull request by benwoo1110**

### CODE REVIEW

File: src/main/java/com/onarandombox/MultiversePortals/MultiversePortals.java [hunk 1]
```diff
@@ -315,7 +314,7 @@ public boolean saveMainConfig() {
             this.MVPConfig.save(new File(this.getDataFolder(), "config.yml"));
             return true;
         } catch (IOException e) {
-            this.log(Level.SEVERE, "Failed to save Portals config.yml.");
+            Logging.info("Failed to save Portals config.yml.");
```
Comment (nicegamer7) [inline src/main/java/com/onarandombox/MultiversePortals/MultiversePortals.java:317]: Was this change intentional?

File: src/main/java/com/onarandombox/MultiversePortals/MultiversePortals.java [hunk 2]
```diff
@@ -325,7 +324,7 @@ public boolean savePortalsConfig() {
             this.MVPPortalConfig.save(new File(this.getDataFolder(), "portals.yml"));
             return true;
         } catch (IOException e) {
-            this.log(Level.SEVERE, "Failed to save Portals portals.yml.");
+            Logging.info("Failed to save Portals portals.yml.");
```
Comment (nicegamer7) [inline src/main/java/com/onarandombox/MultiversePortals/MultiversePortals.java:327]: Same question as above.

**Review by nicegamer7 — COMMENTED** [review]
There seems to be a mistake, unless you meant to change the log level.

File: src/main/java/com/onarandombox/MultiversePortals/MultiversePortals.java (hunk 1 above)
Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiversePortals/MultiversePortals.java:317]: ahh probably a mistake, had to change so many haha

**Review by nicegamer7 — APPROVED** [review]
Looks good 👍.
